absl/strings/numbers.h

```diff
@@ -25,8 +25,12 @@
 #define ABSL_STRINGS_NUMBERS_H_
 
 #ifdef __SSE4_2__
```

Comment:
It doesn't look like __SSE4_2__ is defined by MSVC, so I don't think this is the correct fix. Are you trying to define __SSE4_2__ yourself under MSVC?

Reply:
I think what Eigen3 and Tensorflow do is they assume SSE4.2 is enabled if AVX is enabled since MSVC does define `__AVX__` when `/arch:AVX` flag is used. Not the best fix, but it is safer.

Reply:
I'm investigating whether we can get away with using immintrin.h instead, which is portable to MSVC and I hope exposes all the intrinsics we use here.

Reply:
> It doesn't look like __SSE4_2__ is defined by MSVC
Don't know why, but its defined

Reply:
> It doesn't look like `__SSE4_2__` is defined by MSVC,

But there are no "standart" saying __SSE4_2__ will be never defined by MSVC.